mps_youtube/main.py

```diff
@@ -3130,11 +3150,11 @@ def play_range(songlist, shuffle=False, repeat=False, override=False):
         if returncode == 42:
             n -= 1
 
-        elif returncode == 43:
-            break
+        elif returncode in (43, None):
```

Comment:
No, playsong returns None for several errors. However, I notice that this will not properly handle returncodes 1-3, which should be interpreted as an order to go to the next song.